package.json

```diff
@@ -22,6 +22,7 @@
     "build:clean": "yarn clean && yarn build",
     "build:tsc": "tsc --build tsconfig.build.json",
     "build:post-tsc": "yarn workspaces foreach --parallel --topological --verbose run build:post-tsc",
+    "build:post-tsc:ci": "yarn workspaces foreach --parallel --topological --verbose --exclude root --exclude \"@metamask/snaps-simulator\" --exclude \"@metamask/snaps-execution-environments\" run build:post-tsc",
```

Comment:
If we don't run `snaps-execution-environment` we won't build the LM bundles? Is that intentional? Doesn't that break releases?

Reply:
It's built in the prepare step. I don't think it needs to be built twice? For releases we run `yarn build`, which runs `yarn build:tsc` and `yarn build:post-tsc` (non-`:ci`), so it's built for releases anyway.